tasks/cockpit-tasks-webhook.yaml

```diff
@@ -17,6 +17,19 @@ items:
           infra: webhook
       spec:
         containers:
+          - name: amqp
+            image: cockpit/amqp
+            ports:
+            - containerPort: 5671
+              protocol: TCP
+              name: amqp
+            - containerPort: 5672
+              protocol: TCP
+              name: amqp
```

Comment:
Why does this need two ports? With the same name even?

Reply:
this is incorrect, and I plan on only exposing 5671